### PR TITLE
Fix `highlight-closing-prs-in-open-issues` feature

### LINKED ISSUE #2455: ` highlight-closing-prs-in-open-issues` is broken
```console
❌ highlight-closing-prs-in-open-issues
content.js:2847 TypeError: Cannot read property 'querySelector' of null
    at MutationObserver.highlight_closing_prs_in_open_issues_add (content.js:5738)
    at observeEl (content.js:2735)
    at highlight_closing_prs_in_open_issues_init (content.js:5751)
    at features_run (content.js:2842)
    at content.js:2889
    at content.js:2804
    at onAjaxedPagesRaw (content.js:2799)
```

### PR DESCRIPTION
**Pull request by chelseyong**
Closes #2455. The feature was not working because the GitHub HTML has been updated.

# Test

To see the PR/commit button beside the issue's status:

- Commit that closes issue:
https://github.com/chelseyong/EPP_Git_Solo_Demo/issues/1

- Pull request that closes issue:
https://github.com/sindresorhus/refined-github/issues/2455

- 2 pull requests that closes issue (no effect):
https://github.com/chelseyong/EPP_Git_Solo_Demo/issues/6

- Pull request that closes another pull request:
https://github.com/chelseyong/EPP_Git_Solo_Demo/pull/2

### CODE REVIEW

File: source/features/highlight-closing-prs-in-open-issues.tsx [hunk 1]
```diff
@@ -10,20 +10,26 @@ function add(): void {
...
+		let ref = null;
+		let link = null;
+		if (isPR) {
+			ref = infoBubble
+				.closest('.TimelineItem-body')!
+				.querySelector("a span")!;
+			link = ref.closest("a")!.getAttribute("href")!;
+		} else {
+			ref = select(".commit-message .issue-keyword a")!;
+			link = ref.getAttribute("href")!;
```
[the remark this answers is not in the document]
Comment (chelseyong) [reply source/features/highlight-closing-prs-in-open-issues.tsx]: Changed. Can I ask why though?

[the remark this answers is not in the document]
Comment (fregante) [reply source/features/highlight-closing-prs-in-open-issues.tsx]: It should be `ref.href`, it's shorter than `ref.getAttribute("href")`

**Comment by fregante** [comment]
You're making changes without thinking. `isCommit` is the opposite of "is commit"

**Comment by fregante** [comment]
I think this feature is not worth fixing at this time. GitHub implemented the feature (https://github.com/sindresorhus/refined-github/issues/2372) so we should just move that data to appear _our way_



GitHub doesn't show commits in the header, but it's not a big deal IMHO. Can you change this PR to make that change? The code will be completely different, using `element-ready` instead of `observe-el`

**Comment by fregante** [comment]
> The code will be completely different

I'm closing this PR for now. Feel free to implement the suggestion above though! :)

**Pull request closed without merging**